docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -59,3 +59,38 @@ The `use_refresh_token` configuration must be used in conjunction with the `acce
 
 The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.2.
 {{% /admonition %}}
+
+### OAuth role mapping enforcement
+
+<!-- AuthNZ -->
+
+This change impacts `GitHub` OAuth, `Gitlab` OAuth, `Okta` OAuth and `Generic` OAuth.
+
+Currently if no organization role mapping is found for a user when connecting via OAuth, Grafana doesn’t update the user’s organization role.
+
+With Grafana 10.1, on every login, if the role_attribute_path property does not return a role, then the user is assigned the role specified by the auto_assign_org_role option or the default role for the organization, by default, Viewer.
+
+To avoid overriding manually set roles, enable the skip_org_role_sync option in the Grafana configuration for your OAuth provider before the user logs in for the first time.
```

Comment:
Do we mean a specific user or any user?
```suggestion
To avoid overriding manually set roles, enable the `skip_org_role_sync` option in the Grafana configuration for your OAuth provider before the user logs in for the first time.
```